app/models/asset.rb

```diff
@@ -48,7 +48,7 @@ class Asset
     end
 
     after_transition to: :clean do |asset, _|
-      asset.delay.save_to_cloud_storage
+      SaveToCloudStorageWorker.perform_async(asset)
```

Comment:
Did you discover how Sidekiq stores the parameters passed to `perform_async`? Is there any downside of passing the model instance vs its ID?

Reply:
Ignore my previous comment - I've just seen the next commit where you change this to pass the asset ID.